sdks/go/sdk.go

```diff
@@ -39,11 +41,29 @@ type SDK struct {
 	health sdk.SDK_HealthClient
 }
 
+func port() int {
+	portStr := os.Getenv("AGONES_SDK_GRPC_PORT")
+	if portStr == "" {
+		// Environment variable is not set; use the default port.
+		return defaultPort
+	}
+	p, err := strconv.Atoi(portStr)
+	if err != nil {
+		// Environment variable cannot be parsed; use the default port.
```

Comment:
Should we log a warning here?

Reply:
Do you mean printing to stdout (like in the `WatchGameServer` function? There isn't a logger defined in this package.

Reply:
I added some print statements to stderr since that seems like a better place to put them than stdout.

Reply:
Ah yes, we don't have a logger here. This makes sense :+1: